src/Store/Actions.js

```diff
@@ -37,14 +37,22 @@ export const fetchCveDetails = (cveId, params) => ({
   payload: () => api.fetchCveDetails(cveId, params),
 });
 
-export const fetchClusterDetailTable = (clusterId, params) => ({
-  type: ActionTypes.FETCH_CLUSTER_DETAIL_TABLE,
+export const fetchClusterCveTable = (clusterId, params) => ({
+  type: ActionTypes.FETCH_CLUSTER_CVES_TABLE,
   meta: {
     timestamp: new Date(),
   },
   payload: () => api.fetchClusterCves(clusterId, params),
 });
 
+export const fetchClusterImagesTable = (clusterId, params) => ({
+  type: ActionTypes.FETCH_CLUSTER_IMAGES_TABLE,
+  meta: {
+    timestamp: new Date(),
```

Comment:
For my understanding: why do we need to pass in timestamp?

Reply:
This is also discussed in Technical overview of this app:

https://docs.engineering.redhat.com/display/RHIF/Vulnerability+for+OpenShift#VulnerabilityforOpenShift-Preventingactionraceconditions